src/SFA.DAS.EmployerApprenticeshipsService.Application/Commands/RemoveLegalEntity/RemoveLegalEntityCommandValidator.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using SFA.DAS.EAS.Application.Validation;
+using SFA.DAS.EAS.Domain.Data.Repositories;
+using SFA.DAS.EAS.Domain.Interfaces;
+using SFA.DAS.EAS.Domain.Models.EmployerAgreement;
+using SFA.DAS.EAS.Domain.Models.UserProfile;
+
+namespace SFA.DAS.EAS.Application.Commands.RemoveLegalEntity
+{
+    public class RemoveLegalEntityCommandValidator : IValidator<RemoveLegalEntityCommand>
```

Comment:
I know this goes against how every validator in DAS has been  written but...

Could/should this class not be internal?  It'll never be instantiated outside of the application layer.  As you've tested through the command handler anyway (which is the right way to do it, IMO) it would be a straight forward change.  Obviously, it'd be inconsistent so feel free to ignroe. ;)